wgpu-types/src/lib.rs

```diff
@@ -326,12 +326,14 @@ bitflags::bitflags! {
         const MAPPABLE_PRIMARY_BUFFERS = 1 << 16;
         /// Allows the user to create uniform arrays of textures in shaders:
         ///
-        /// eg. `uniform texture2D textures[10]`.
+        /// eg. `var textures: array<texture_2d<f32>, 10>` (WGPU), `uniform texture2D textures[10]`
+        /// (GLSL).
```

Comment:
Four nits (sorry for the amount on this one lol), first lets put these on separate lines, second /s/wgpu/wgsl/, third /s/eg/ex/, fourth /s/array/binding_array/ - all binding arrays need to use the binding_array type

```suggestion
        /// ex. 
        /// `var textures: binding_array<texture_2d<f32>, 10>` (WGSL)
        /// `uniform texture2D textures[10]` (GLSL)
```

Reply:
> second /s/wgpu/wgsl

Oops. Acronyms are hard, haha.

> third /s/eg/ex

Should I update the other occurrences of 'eg.' in the `Features` documentation? I see eight others.

> fourth /s/array/binding_array

Interesting; good to know.

Reply:
> Should I update the other occurrences of 'eg.' in the Features documentation? I see eight others

Yeah that'd be great!

Reply:
Oh another subtle detail here, the first example should have two spaces after it so that markdown keeps it the same paragraph, but on different lines.

Reply:
Should there be a double-space line break after 'ex.' as well, or no?